_posts/2019-01-04-nextjs.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: Create Universal Apps Using Next.js
+date: 2019-03-18 10:00:00 +0200
+categories: javascript
+author_names: ["Andreas Geier", "Sergii Paryzhskyi"]
+read_time : 10
+show_related_posts: false
+square_related:
+feature_image: posts/2019-01-04-nextjs/poster.png
+---
+
+Let's talk at first about libraries for creating universal applications. On Github you can find a large number of projects implementing the idea of universal web applications. However, all these projects have common drawbacks:
+
+- missing functionality such as server side rendering (SSR) or i.e. hot reloading
+- these are projects for quick start, not a library
```

Comment:
You can drop `these are projects`, as this is already in context, and simply go with: `they are designed for quick start, ...`.